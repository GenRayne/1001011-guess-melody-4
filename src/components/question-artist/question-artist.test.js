import React from 'react';
import renderer from 'react-test-renderer';
import QuestionArtist from './question-artist.jsx';
import questions from '../../mocks/test-questions.js';

const noopHandler = () => {};

describe(`render QuestionArtist`, () => {
  it(`renders QuestionArtist`, () => {
    const tree = renderer
      .create(
          <QuestionArtist
            question={questions[1]}
            onAnswer={noopHandler}
            renderPlayer={noopHandler}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
