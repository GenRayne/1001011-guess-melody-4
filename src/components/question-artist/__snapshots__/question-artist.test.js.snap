// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render QuestionArtist renders QuestionArtist 1`] = `
Array [
  <h2
    className="game__title"
  >
    Кто исполняет эту песню?
  </h2>,
  <div
    className="game__track"
  >
    <div
      className="track"
    >
      <button
        aria-label="Pause"
        className="track__button track__button--pause"
        disabled={true}
        onClick={[Function]}
        title="Pause"
        type="button"
      />
      <div
        className="track__status"
      >
        <audio />
      </div>
    </div>
  </div>,
  <form
    className="game__artist"
  >
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        id="answer-1"
        name="answer"
        onChange={[Function]}
        type="radio"
        value="artist-1"
      />
      <label
        className="artist__name"
        htmlFor="answer-1"
      >
        <img
          alt="John Snow"
          className="artist__picture"
          src="https://api.adorable.io/avatars/128/10"
        />
        John Snow
      </label>
    </div>
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        id="answer-2"
        name="answer"
        onChange={[Function]}
        type="radio"
        value="artist-2"
      />
      <label
        className="artist__name"
        htmlFor="answer-2"
      >
        <img
          alt="Jack Daniels"
          className="artist__picture"
          src="https://api.adorable.io/avatars/128/16"
        />
        Jack Daniels
      </label>
    </div>
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        id="answer-3"
        name="answer"
        onChange={[Function]}
        type="radio"
        value="artist-3"
      />
      <label
        className="artist__name"
        htmlFor="answer-3"
      >
        <img
          alt="Jim Beam"
          className="artist__picture"
          src="https://api.adorable.io/avatars/128/24"
        />
        Jim Beam
      </label>
    </div>
  </form>,
]
`;
